docs/development/extensions-contrib/iceberg.md

```diff
@@ -0,0 +1,121 @@
+---
+id: iceberg 
+title: "Iceberg"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+## Iceberg Ingest Extension
```

Comment:
```suggestion
## Iceberg Ingest extension
```

Reply:
I don't think this heading is necessary. Since the topic is on Iceberg, consider introducing the feature first and then talk about the extension as a means of enabling the feature. For example:

Apache Iceberg is an open table format for huge analytic datasets. [Iceberg input source](../../ingestion/input-sources.md#iceberg-input-source) lets your ingest data stored in the Iceberg table format into Apache Druid. To  enable Iceberg input source, add the `druid-iceberg-extensions` extension to the list of extensions. See [Loading extensions](../../configuration/extensions.md#loading-extensions) for more information.

Iceberg manages most of its metadata in metadata files in the object storage. In some cases, it uses a metastore to manage a certain amount of metadata.